landingzones/views_api.py

```diff
@@ -227,6 +233,60 @@ def perform_create(self, serializer):
             raise APIException('{}{}'.format(ex_msg, ex))
 
 
+class ZoneUpdateAPIView(
+    ZoneModifyMixin, SODARAPIGenericProjectMixin, UpdateAPIView
+):
+    """
+    Update a landing zone description and user message.
```

Comment:
Blank lines are missing from this docstring. Remember, they affect how the docstring will be rendered in Sphinx. Make sure to build the docs and check that everything seems correct.